src/icons/BurnerWalletDivider.tsx

```diff
@@ -0,0 +1,206 @@
+export default function () {
+  return (
+    <svg
+      height="23"
+      viewBox="0 0 310 31"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M165.619 14.5419C166.032 15.1216 166.032 15.8784 165.619 16.4581C164.177 18.4804 160.33 23 154.75 23C149.17 23 145.323 18.4804 143.881 16.4581C143.468 15.8784 143.468 15.1216 143.881 14.5419C145.323 12.5196 149.17 8 154.75 8C160.33 8 164.177 12.5196 165.619 14.5419Z"
+        stroke="#EFECD6"
```

Comment:
You can put strokeColor and strokeWidth into `const commongStroke = classnames(...)`